docker/Dockerfile

```diff
@@ -153,7 +156,7 @@ RUN set -ex; \
     echo "cache_dir = '/zebrad-cache'"; \
     echo "[tracing]"; \
     echo "endpoint_addr = '0.0.0.0:3000'"; \
-  } > "zebrad.toml"
+  } > "${ZEBRA_CONF_PATH}"
```

Comment:
Do we need a test that checks that $ZEBRA_CONF_PATH works?
Do we already have a test that makes sure this default config works with the latest Zebra version, or do we need to add that test?

Reply:
We added some config tests in #4684, so another option is adding this config to that directory.

Reply:
I agree we might want to have a test to check `$ZEBRA_CONF_PATH` works, just as [oxarbitrage](https://github.com/oxarbitrage) added here [tests(config): Upgrade config file tests](https://github.com/ZcashFoundation/zebra/pull/5112#top)
#5112

Reply:
I'm not the best one to create the description of this ticket, is this won't be done in this PR (or in the near future)

Reply:
I created ticket #5168